src/formatting-strings.md

```diff
@@ -0,0 +1,38 @@
+# Format Strings Using `%q`
+
+Whenever formatting messages that contain a string component via `fmt`, use `%q` instead of `%s`. This will wrap the specified string in quotes, helping it stand out from the rest of the error message. More importantly, if the string is empty, it will provide a more helpful error message.
```

Comment:
> [...] helping it stand out from the rest of the error message.

> More importantly, if the string is empty, it will provide a more helpful error message.

I'd either generalize these slightly or remove the word "error" (this guidance can apply to more than just errors). Maybe something like:

> More importantly, if the string is empty or has its own formatting, the result
> will be clearer and more consistent.